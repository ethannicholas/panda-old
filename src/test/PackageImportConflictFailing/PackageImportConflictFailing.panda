package example

class String {
    constructor() {
        Console.writeLine("success")
    }
}

package differentPackage

uses example.*

class Test {
    @class
    method main() {
        new String()
    }
}