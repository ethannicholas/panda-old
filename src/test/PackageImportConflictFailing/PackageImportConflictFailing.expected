error: PackageImportConflictFailing.panda:16:13: class 'String' is ambiguously referenced, both panda.core.String and example.String match
1 error
